test/verifiedTests/design/testMultiProductionEnvelope.m

```diff
@@ -0,0 +1,38 @@
+% The COBRAToolbox: testmultiProductionEnvelope.m
+%
+% Purpose:
+%     - test the multiProductionEnvelope function
+%
+% Authors:
+%     - Jacek Wachowiak
+global CBTDIR
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which('testMultiProductionEnvelope'));
+cd(fileDir);
+
+% test variables
+model = readCbModel([CBTDIR filesep 'test' filesep 'models' filesep 'ecoli_core_model.mat']);
+biomassRxn = model.rxns(1);
+target = model.rxns(2);
+deletions = model.rxns(3);
+deletions2 = model.genes(3);
+
```

Comment:
The same here, what is the reason for this choice? There are genes which influence the biomass and by-product production, but the chosen ones seem quite random.